docs/tractus-x-release.md

```diff
@@ -0,0 +1,21 @@
+# Tractus-X Release
+
+## Artifacts
+
+- Umbrella Helm Chart(s)
+- Changelog (overarching feat)
+
+## Versioning
+
+- Semantic Versioning
+
+## Process
+
+<!-- How are products (considered to be) integrated in the Tractus-X release -->
+- Prior to release, a Tractus-X umbrella Chart release candidate branch is created
+- Producs issue PRs with their release candidate
+
```

Comment:
Good input. I'll see if I can form that to specific sections once I finally find some time for it again. I will try to avoid some of the terms you used though. Some are not (yet) known in Eclipse Tractus-X and would need too much explanation. I.e.:
`cxOS` -> If there is nothing provided by the association that we can reference, we we would need to re-specify, which is error prone and might diverge from their definiition.
`onion rings` -> meaning? relevance? 
`carry-over versions` -> not sure what is meant with that
`boundary conditions for bundles` -> what are these conditions, who defines them, where can I read about it?

Reply:
cxOS may not yet be fully specified-out, but is mentioned and described by the association in the following zwo (maybe more) documents. So referencing without re-specification should be ok.
https://catena-x.net/fileadmin/_online_media_/231006_Whitepaper_Tractus-X.pdf (https://catena-x.net/fileadmin/_online_media_/231006_Whitepaper_Tractus-X.pdf)
https://catena-x.net/fileadmin/_online_media_/231006_Whitepaper_LifeCycleManagement.pdf (https://catena-x.net/fileadmin/_online_media_/231006_Whitepaper_LifeCycleManagement.pdf)

Reply:
the pictogram of "onion rings" shall be published near term, also via association, with regards to test management. will add the link here as soon as available. @danielmiehle any additional info?

Reply:
carry-over: in every quarterly TX release, we shall expect evolutions (new versions) of some cxOS components,
but also some other components may NOT face a change/up-revision in the given iteration. Nevertheless those components are still essential parts of the bundle.
This is what I meant with "carry over versions"

Reply:
text suggestion @stephanbcbauer & @RolaH1t 21-Mar:
Once feature work on product level is initiated, the quality assurance of the foreseen release package runs in parallel. The vehicle of Umbrella helm charts is used to continuously test pre-defined product bundles. Automatically and environment-less.
Reference onion rings: (add URL)

Note: In every quarterly TX release, we shall expect evolutions (new versions) of some components (see "product_release.md"). But also some other components may NOT face a change/up-revision in the given iteration. Nevertheless those components are still essential parts of the bundle.

Example for cxOS (https://catena-x.net/fileadmin/_online_media_/231006_Whitepaper_Tractus-X.pdf (https://catena-x.net/fileadmin/_online_media_/231006_Whitepaper_Tractus-X.pdf):
A configured umbrella helm chart starts-up all relevant components. No matter if carry-over versions of a previous TX release or evolutions of the current iteration. User Journey tests (e2e) are automatically performed - customized for the respective bundle of components and within given boundary conditions.
Any findings ale looped back to the product development teams.
Product teams are obliged to fix any defects which prevent successful testing of the bundle.
A bundle is ready for Tractus-X release, once product feature work is completed for all components and e2e-testing was successful without critical findings.

Additionally, manual e2e tests may apply. Environment and execution tbd.

Reply:
I get the idea of this description, but to me it almost seems like this is talking a different language, than I would expect from this process definition. 
In commit [c33fcc8](https://github.com/eclipse-tractusx/sig-release/pull/519/commits/c33fcc82d867d7bbd776cd09e20130344b0129da) I started a process definition, that I think could make sense. It is not finished though and is also just a proposal, that we must discuss in the comitter round.

As general suggestion for wording, I would avoid terms, that are unknown, undefined or not commonly used.
I.e. carry-over, evolution, up-revision.
I would also the "what is NOT happening" statements. Instead of "maybe there are no changes, but we still need it", I would just state, that we are testing all components in the version, that is planned to be included in the release